scripts/update/Updater.php

```diff
@@ -527,6 +527,17 @@ public function update($initialVersion)
 
         $this->skip('12.23.0', '12.24.1');
 
+        if ($this->isVersion('12.24.1')) {
+            $file = __DIR__ . DIRECTORY_SEPARATOR .
```

Comment:
Why we should use DIRECTORY_SEPARATOR here? It may sense to use it for extract part from a real path on Windows, but for composing Linux-like separator ("/") will work well on both platforms.

Another approach, using a straight path with __DIR__ '/path/to/file' allows quickly jump to a real file in the IDE and see highlights when typing the path.

Reply:
We shouldn't. It's just a slightly modified copy of a similar block up above ;)